app/templates/components/component-tree-item.hbs

```diff
@@ -1,28 +1,30 @@
 <div
-  style={{labelStyle}}
-  class="component-tree-item {{if item.view.isComponent "component-tree-item--component"}} {{if (eq pinnedObjectId item.view.objectId)"component-tree-item--selected"}}"
-  {{action inspect item.view.objectId}}
+  class="component-tree-item {{if @item.view.isComponent "component-tree-item--component"}} {{if @selected "component-tree-item--selected"}} {{if @highlighted "component-tree-item--highlighted"}}"
+  onmouseenter={{@eventMouseEnter}}
+  onmouseleave={{@eventMouseLeave}}
+  style={{component-indentation @item.hasChildren @item.parentCount}}
+  {{action inspect}}
 >
-  {{#if item.hasChildren}}
+  {{#if @item.hasChildren}}
     {{ui/disclosure-triangle
       class="component-tree-item__expand"
+      expanded=@item.expanded
       title="Click to toggle (alt/option click to toggle with children)"
-      expanded=item.expanded
-      toggle=(action toggleExpanded)
+      toggle=(action @toggleExpanded)
     }}
   {{/if}}
 
   <code
-    class="component-tree-item__tag {{if item.view.isComponent "component-tree-item__bracket"}}"
+    class="component-tree-item__tag {{if @item.view.isComponent "component-tree-item__bracket"}}"
   >
-    {{item.view.name}}
+    {{@item.view.name}}
   </code>
 
-  {{#if item.view.isComponent}}
+  {{#if @item.view.isComponent}}
     <button
       class="component-tree-item__action js-scroll-into-view"
       title="Scroll into view"
-      {{action scrollToElement item.view.objectId bubbles=false}}
```

Comment:
Shouldn't we still pass the objectId here? If not, how does it know what to scroll to?

Reply:
https://github.com/emberjs/ember-inspector/pull/984/commits/87c19898a1d8dfc89794956107327eb6eec7a5dc#diff-6a9fa9ada69626e649ae996c3753adbbR7

^ In the `component-tree-item` template, the `scrollToElement` attribute has the objectId already rolled into it from the invocation of `<ComponentTreeItem />`

Reply:
Ahhh, gotcha 👍